src/server/src/test/java/io/cassandrareaper/management/http/HttpCassandraManagementProxyTest.java

```diff
@@ -330,6 +330,60 @@ public void testGetCompactions() throws Exception {
     verifyNoMoreInteractions(mockClient);
   }
 
+  @Test
+  public void testGetLocalEndpoint() throws Exception {
+    DefaultApi mockClient = Mockito.mock(DefaultApi.class);
+    EndpointStates states = new EndpointStates();
+    states.addEntityItem(ImmutableMap.of(
+        "ENDPOINT_IP", "10.0.0.1",
+        "IS_LOCAL", "false"
+    ));
+    states.addEntityItem(ImmutableMap.of(
+        "ENDPOINT_IP", "10.0.0.2",
+        "IS_LOCAL", "true"
+    ));
+    when(mockClient.getEndpointStates()).thenReturn(states);
+
+    HttpCassandraManagementProxy proxy = mockProxy(mockClient);
+    String localEndpoint = proxy.getLocalEndpoint();
+
+    assertThat(localEndpoint).isEqualTo("10.0.0.2");
+    verify(mockClient).getEndpointStates();
+
+    String untranslatedHost = proxy.getUntranslatedHost();
```

Comment:
question: do we still need `getUntranslatedHost` if it's just a pass-through to `getLocalEndpoint`?

Reply:
Yes, because once address translators are added, these two will do something different.

Reply:
Does "untranslated" refer to the internal IP or the external one?

Reply:
I assume the external one is "translated" from the internal one. So the external one - @adejanovski can you advise?

Reply:
OK, I think the two methods are the same, but I will leave `getUntranslatedHost()` with the TODO comment until this gets tested with address translation.